packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/tests/index.test.js

```diff
@@ -49,17 +61,19 @@ const store = createStore(rootReducer, {
 
 const setup = (props) =>
   render(
```

Comment:
Can you please use `wrapper({ children })` here: https://testing-library.com/docs/react-testing-library/api/#wrapper

Reply:
done thanks